keras_nlp/models/bert.py

```diff
@@ -251,6 +266,152 @@ def get_config(self):
         return config
 
 
+TOKENIZER_DOCSTRING = """Bert tokenizer with pre-trained vocabularies.
+
+This layer will tokenize inputs as a specialized
+`keras_nlp.tokenizers.WordPieceTokenizer` which supports loading pre-trained
+vocabularies.
+
+If `pack_inputs` is True, this layer will accept a tuple of (possibly batched)
+inputs, tokenize each input, truncate them according to the `trucator` strategy,
+and pack with them appropriate `"[CLS]"` and `"[PAD]"` tokens. The output of the
+layer will be a dictionary with three keys `"token_ids"`, `"segment_ids"`, and
+`"padding_mask"`. The output can be used directly by a Bert model.
+
+If `pack_inputs` is False, this layer acts only as a raw tokenizer. Input should
+be a single (possibly batched) input, and output will be a `tf.RaggedTensor` of
+token ids. This is useful when inputs need to be packed in a custom fashion,
+but output will need to be further processed before being passed to a Bert
+model.
+
+Args:
+    vocabulary: One of a list of vocabulary terms, a vocabulary filename, or
+        the name of the pre-trained vocabulary. For a pre-trained vocabulary,
+        `vocabulary` should be one of {names}, and should match the `weights`
+        parameter of any pre-trained Bert model.
+    lowercase: If `True`, input will be lowercase before tokenization. If
+        `vocabulary` is set to a pre-trained vocabulary, this parameter will
+        be inferred.
+    pack_inputs: If `True`, the layer will accept multiple inputs, pack them
+        into a single sequence, and return a dictionary of
+    sequence_length: The length of the packed inputs. Only used if
+        `pack_inputs` is True.
+    truncator: The algorithm to truncate a list of batched segments to fit
+        within `sequence_length`. Only used if
+        `pack_inputs` is True. The value can be either `round_robin` or
+        `waterfall`:
+            - `"round_robin"`: Available space is assigned one token at a
```

Comment:
Does this render properly on keras.io?

Reply:
No :(, is there a good way to do this in a keras.io friendly way?

We have this problem elsewhere it looks like, take a look at `output_mode` for `StringLookup`.
https://www.tensorflow.org/api_docs/python/tf/keras/layers/StringLookup
https://keras.io/api/layers/preprocessing_layers/categorical/string_lookup/